src/io/fasta.rs

```diff
@@ -476,23 +481,38 @@ impl<W: io::Write> Writer<W> {
 /// A FASTA record.
 #[derive(Default)]
 pub struct Record {
-    header: String,
-    seq: String,
+    pub id: String,
+    pub desc: String,
+    pub seq: String,
```

Comment:
As below for fastq: desc should be an option type, and preferably the attributes should not be public.